### DISCUSSION TITLE
is there any work chance for roda/sequel ?

### DISCUSSION THREAD

**Discussion by zw963** [jeremyevans/roda#234] [General]
Don't know if this question suitable for discussion here, anyway, if not be apt to, just ignore it.

briefly introduce my career, I come from China,, i am not a computer major, full self-teach, and finished reading  `Ruby programming language` systematically, before i start my programming career probably 2011. In the last 10 years, most of time, as of other ruby programmer, I work with web + ruby on rails, and write ruby daily, and work remotely recent three years,  i love ruby, but ruby on rails was not so favorite, as leas, version before 6, i write several gem too, and many ruby code locally serve for only myself, i do write many bash script, emacs lisp script, all of those serve myself for work efficiency, in short, i make many wheels too.
 
Okay, my personal introduce is enough anyway, more detail can give me a email, for now, i love many philosophy of roda/sequel stack, Lightweight, plugin system, free configuration, less gem is better, less code is better, no too much magic, no too much core_ext, etc ...  i love the capability those awesome gems can give me,  make me free, unconstraintly, i am working on a roda/sequel project for now, very happen with this.

But, though ruby on rails work changes very popular in china, but, roda/sequel stacks was (almost) none, even, no one knew it, sinatra is more popular than roda, and always prefer ActiveRecord over Sequel.

so, i want to if there have work chance for roda/sequel or whatever ruby work chance (non-rails first anyway).

BTW:  i am very self-confidence with my ruby skill, in fact, i am a TDD lover, because this reason, i work on ThoughtWorks China for  2013-2014 year.

**Comment by jeremyevans** [comment]
I think you'll find that's true in most places, not just China.  The job market for Rails is far larger than Ruby but not Rails.  This is one of the main benefits of using Rails, the other being it's easier to find people and libraries that work with Rails.

There are certainly for-profit and non-profit companies as well as government departments using Roda and Sequel for production applications, though most aren't very vocal about it.   Here's one recent example (from what I can guess based on some brief research, the company involved is Prime Trust): https://mobile.twitter.com/_mculp/status/1474833652764884996

**Reply by zw963** [reply]
If we can have one place,  public/search roda/sequel job there,  that maybe help a lot for roda/sequel ecology.

anyway,  such high quality gems, many rubyist even don't know it.

**Reply by jeremyevans** [reply]
I certainly encourage companies that are looking for employees for Roda/Sequel applications to post on the respective GitHub Discussions and/or Google Group.
